### PR TITLE
change the structure of describing environment variables on how to run docs.

### PR DESCRIPTION
**Pull request by ahmetoz**
On one of the review: https://github.com/commercetools/commercetools-adyen-integration/pull/918#discussion_r808983409 we had a discussion and found out the existing documentation creates confusion, as it's grouped in a way that you can not understand what is the required values and optional ones, also the given examples with the values are defined with the not default values as it's better to suggest to use default values as it might be the better choice: https://github.com/commercetools/commercetools-adyen-integration/pull/918#discussion_r808985460

### CODE REVIEW

File: extension/docs/HowToRun.md [hunk 1]
```diff
@@ -43,113 +43,60 @@ Extension module requires 1 environment variable to start. This environment vari
...
-| `addCommercetoolsLineItems`  | Boolean attribute. If set to **true**, integration will add lineItems to payment methods that require lineItems on `makePaymentRequest`. Payment method types that requires [lineItems](https://docs.adyen.com/api-explorer/#/CheckoutService/latest/payments__reqParam_lineItems): `klarna`, `affirm`, `afterpay`, `afterpaytouch`, `klarna`, `ratepay`, `facilypay`, `clearpay`, `grabpay`, `paybright`, `pix`, `zip`. | NO       | false                                                                                                                        |
+> Note that, multiple child attributes can be provided in the `adyen` and `commercetools` attribute group. Other configurations can be set as direct child attributes in `ADYEN_INTEGRATION_CONFIG`.
```
Comment (lojzatran) [inline extension/docs/HowToRun.md:65]:
```suggestion
> Note that multiple child attributes can be provided in the `adyen` and `commercetools` attribute group. Other configurations can be set as direct child attributes in `ADYEN_INTEGRATION_CONFIG`.
```

https://forum.english.best/t/comma-or-no-comma-after-note-that/24557, https://www.englishforums.com/English/CommaAfterNoteThat/jlkb/post.htm

File: extension/docs/HowToRun.md [hunk 2]
```diff
@@ -43,113 +43,60 @@ Extension module requires 1 environment variable to start. This environment vari
...
+
+#### Preparing the credentials
+
+- Adyen credentials:
+  - For **test environment** follow the official Adyen [get started guide](https://docs.adyen.com/checkout/get-started) to set up your **test account**, get your API key.
+  - For **live environment** follow the official Adyen [documentation](https://docs.adyen.com/user-management/get-started-with-adyen#step-2-apply-for-your-live-account) for details.
+- commercetools project credentials:
+  - If you don't have the commercetools OAuth credentials,[create a commercetools API Client](https://docs.commercetools.com/getting-started.html#create-an-api-client).
+    - Note that, extension module requires `manage_payments, view_orders` [scopes](https://docs.commercetools.com/http-api-scopes) for the integration and `manage_types, manage_extensions` [scopes](https://docs.commercetools.com/http-api-scopes) for setting up required resources.
```
Comment (lojzatran) [inline extension/docs/HowToRun.md:74]:
```suggestion
    - Note that extension module requires `manage_payments, view_orders` [scopes](https://docs.commercetools.com/http-api-scopes) for the integration and `manage_types, manage_extensions` [scopes](https://docs.commercetools.com/http-api-scopes) for setting up required resources.
```

File: extension/docs/HowToRun.md [hunk 3]
```diff
@@ -43,113 +43,60 @@ Extension module requires 1 environment variable to start. This environment vari
...
+| `other`         | `basicAuth`                  | Boolean attribute to enable/disable basic authentication to prevent unauthorized 3rd-party from accessing extension endpoint                                                                                                                                                                                                                                                                                             | false                                                                                                                                                                                   |
```
Comment (lojzatran) [inline extension/docs/HowToRun.md:93]: Attribute group and `other` needs to be explained more visible and explicit what they mean. I saw there is a small mention about it above `Other configurations can be set as ...`, but I don't think this is enough for people to understand.

Maybe make a small section with name `Attribute group` just above these tables and write there what every group means?

**Review by lojzatran — CHANGES_REQUESTED** [review]

File: extension/docs/HowToRun.md (hunk 3 above)
Comment (ahmetoz) [reply extension/docs/HowToRun.md:97]: How does it look like now: https://github.com/commercetools/commercetools-adyen-integration/pull/938/commits/92c776a76280f35c0658c2dd97eaa69f06fd9601

**Review by lojzatran — APPROVED** [review]

**Pull request merged**